docs/gitbook/reference/interbit/createChains.md

```diff
@@ -1,60 +1,89 @@
 # `createChains()`
 
-An object containing two functions: createChainsFromManifest and createChainsFromConfig
+An object containing two functions: createChainsFromManifest and
+createChainsFromConfig
 
 ## `createChainsFromManifest()`
 
 <div class="tips danger">
   <p><span></span>TODO</p>
-  <p>This function signature and name is set to change in a future release. It will be renamed to createChains and location will be moved into options. Other changes may occur.</p>
+  <p>
+    This function signature and name is set to change in a future
+    release. It will be renamed to `createChains` and its location will
```

Comment:
These backticks inside of the <p> tags end up rendered as backticks. Markdown is not supported inside of HTML tags.

<code>createChains</code> will work, though!